get-changed-packages.ts

```diff
@@ -190,7 +190,7 @@ export let getChangedPackages = async ({
     changedPackages: (packages.tool === "root"
       ? packages.packages
       : packages.packages.filter((pkg) =>
-          changedFiles.some((changedFile) => changedFile.includes(pkg.dir))
+          changedFiles.some((changedFile) => changedFile.startsWith(`${pkg.dir}/`))
```

Comment:
1. `startsWith` is more appropriate than `includes` here for edge cases
2. trailing slash is important to handle cases when 2 packages are in similar directories. I ran into this with `packages/replayio` and `packages/replay`